psdisk/psdisk.c

```diff
@@ -491,6 +477,12 @@ static int psdisk_parseToSave(const char *arg)
 		return -1;
 	}
 
+	if (ptable_typeName(node->part.type) == NULL) {
```

Comment:
Hmm, why check this?

Reply:
When changing libptable's header I only added a list of known partition types (`ptable_knownTypes`, for flexible usage) and `ptable_typeName()`, which can also be used to check whether a given type is a known one instead of iterating over the array. Maybe I should have added a separate function

Reply:
What I mean - why do we want to check if the partition type is a known one _at all_?

Reply:
Well, to align with `image_builder.py`'s behavior, which checks the type